crates/kira/src/manager/backend/cpal/desktop.rs

```diff
@@ -21,27 +21,46 @@ enum State {
 	},
 }
 
+/// Settings for the [`cpal`] backend.
+#[derive(Default)]
+pub struct CpalBackendSettings {
```

Comment:
Can we use the same `CpalBackendSettings` struct regardless of platform? Right now it's defined twice: once for desktop and once for wasm, but the definition is the same for both.